### PR TITLE
add node16 and update lts, latest, and current tagged images

### PR DESCRIPTION
**Pull request by polyomino-24**
I added Node.js v16, which is latest and current version according to [docker hub](https://hub.docker.com/_/node), and moved LTS tag to v14.

### CODE REVIEW

File: npm/cloudbuild.yaml [hunk 1]
```diff
@@ -65,10 +68,12 @@ steps:
...
 - name: 'gcr.io/$PROJECT_ID/npm:node-12.18.3'
   args: ['--version']
-- name: 'gcr.io/$PROJECT_ID/npm:node-14.10.0'
+- name: 'gcr.io/$PROJECT_ID/npm:node-14.17.6'
+  args: ['--version']
+- name: 'gcr.io/$PROJECT_ID/npm:node-16.8.0'
   args: ['--version']
 
-# Test the examples with :latest
+# Test the examples with :latest and :current
```
Comment (polyomino-24) [inline npm/cloudbuild.yaml:76]: I think it would be better to test in LTS version too. What do you think?

**Comment by jordanebelanger** [comment]
> @polyomino-24 Now that Node 16 is LTS, it would be great if we could (1) get the node 16 image in to the registry, and (2) move the LTS tag accordingly.

++ for this, even the documentation on GCP specifies that node 16 is now LTS, come on people

**Comment by marcushultman** [comment]
@jordanebelanger The Node community maintains [official Node.js docker images](https://hub.docker.com/_/node?tab=description), which is likely why Google is not maintaining the same images anymore.

**Comment by arvinddayal** [comment]
#869 should take care of this.

**Pull request closed without merging**